roles/openshift_node/tasks/main.yml

```diff
@@ -36,7 +36,7 @@
 
 - name: Install sdn-ovs package
   action: "{{ ansible_pkg_mgr }} name={{ openshift.common.service_type }}-sdn-ovs{{ openshift_pkg_version | oo_image_tag_to_rpm_version(include_dash=True) }} state=present"
-  when: openshift.common.use_openshift_sdn and not openshift.common.is_containerized | bool
+  when: openshift.common.use_openshift_sdn and not openshift.common.use_flannel | bool and not openshift.common.is_containerized | bool
```

Comment:
@abutcher ptal. If we do this for flannel, we would also have to do it for the nuage flag, and I'm not sure this creates the best interface. It might be confusing for a user to know which "use_<network plugin>" overrides the others.